src/frontend/src/lib/services/address.services.ts

```diff
@@ -89,6 +98,11 @@ const bitcoinMapper: Record<
 	testnet: {
 		addressStore: btcAddressTestnetStore,
 		setIdbAddress: setIdbBtcAddressTestnet
+	},
+	regtest: {
+		addressStore: btcAddressRegtestStore,
+		// No need to store the regtest in the local storage because it's only used locally.
+		setIdbAddress: undefined
```

Comment:
Considering how often you are going to test this locally, maybe it will be worth to save it at some point, to spare calls? Or not necessary for the effort?

Reply:
I wanted to skip storing things in local state.

I also assume we will be storing it in the user profile at some point as discussed in the daily. The extra code is code that we won't have to remove later.

If you have a strong opinion I can add it. i thought it simplified the PR. If I can avoid code I'll avoid it 😄

Reply:
Nono, no strong opinion, just wanted to suggest eheh